django-verdant/rca/static/rca/css/variables.less

```diff
@@ -152,6 +152,7 @@ you would use 'font-weight:bold;', use the '.bold()' mixin instead */
 @color-warm-grey: #b6a2a2;
 @color-middle-column-lines: #d6d6d6;
 @color-tab-grey: darken(@color-pale-grey, 10%);
+@color-icon-grey: #cdcdcd;
```

Comment:
Might this be close enough to the existing `@color-light-grey` to use that instead?

Reply:
removed